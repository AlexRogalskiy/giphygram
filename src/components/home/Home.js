import React from 'react';
import { SearchResultsContainer } from '../../containers/searchResultsContainer/SearchResultsContainer';
import { SearchSummaryContainer } from '../../containers/searchSummaryContainer/SearchSummaryContainer';

export class Home extends React.Component {
  render() {
    return (
      <div>
        <SearchSummaryContainer />
        <SearchResultsContainer />
      </div>
    );
  }
}
