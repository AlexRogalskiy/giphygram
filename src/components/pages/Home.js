import React from 'react';
import { SearchForm } from '../searchForm/SearchForm';
import { GifList } from '../gifList/GifList';

export class Home extends React.Component {
  render() {
    return (
      <div>
        <SearchForm/>
        <GifList/>
      </div>
    );
  }
}
