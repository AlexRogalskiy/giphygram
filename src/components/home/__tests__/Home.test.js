import React from 'react';
import renderer from 'react-test-renderer';
import { Home } from '../Home';

jest.mock(
  '../../../containers/searchResultsContainer/SearchResultsContainer',
  () => 'SearchResultsContainer',
);

jest.mock(
  '../../../containers/searchSummaryContainer/SearchSummaryContainer',
  () => 'searchSummaryContainer',
);

describe('Home', () => {
  it('should be rendered correctly', () => {
    const tree = renderer
      .create(<Home />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
