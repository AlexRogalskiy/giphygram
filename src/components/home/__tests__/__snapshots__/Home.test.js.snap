// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home should be rendered correctly 1`] = `
<div>
  <searchSummaryContainer />
  <SearchResultsContainer />
</div>
`;
